bundles/AdminBundle/src/Controller/Admin/UserController.php

```diff
@@ -272,16 +272,12 @@ public function deleteAction(Request $request): JsonResponse
     /**
      * @Route("/user/update", name="pimcore_admin_user_update", methods={"PUT"})
      *
-     * @param Request $request
-     *
-     * @return JsonResponse
-     *
      * @throws \Exception
      */
     public function updateAction(Request $request): JsonResponse
     {
-        /** @var User|null $user */
-        $user = User\UserRole::getById((int)$request->get('id'));
+        /** @var User|User\Role|null $user */
+        $user = User\UserRole::getById($request->request->getInt('id'));
```

Comment:
Are we sure this is always in the request? 
We had some problems with other changes that the data was not always in the request but sometimes in the query e.g.

Reply:
I looked in the JS files. The id was alwas set in POST data.